azure-simple/packer/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need this license file since the same file is in the root directory?

Reply:
Removed.